test/assert.rb

```diff
@@ -22,7 +22,7 @@ def assertion_string(err, str, iso=nil, e=nil, bt=nil)
   msg += " [#{iso}]" if iso && iso != ''
   msg += " => #{e.cause}" if e && e.respond_to?(:cause)
   msg += " => #{e.message}" if e && !e.respond_to?(:cause)
-  msg += " (mrbgems: #{GEMNAME})" if Object.const_defined?(:GEMNAME)
+  msg += " (#{GEMNAME == 'mruby-test' ? 'core' : "mrbgems: #{GEMNAME}"})"
```

Comment:
`GEMNAME` does not seem to be defined in `bintest`.
Therefore, if an assertion fails, an undefined exception will be raised.

Reply:
Ah, I've forgotten `test/assert.rb` is run on CRuby. I'll fix the issue.

Thank you!